internal/command/machine/run.go

```diff
@@ -629,6 +629,11 @@ func determineMachineConfig(ctx context.Context, initialMachineConf api.MachineC
 		machineConf.Schedule = flag.GetString(ctx, "schedule")
 	}
 
+	if command := flag.GetString(ctx, "command"); command != "" {
+		command := []string{command}
```

Comment:
I think in this case we want to use a shell parser like `shlex.Parse()`, wdyt?

Reply:
Right, good catch. I'll circle back to this on Monday and will look into this--new to Go and not familiar with that package but should be simple enough. Thanks!